### ISSUE TITLE
is this a dead project?

### ISSUE THREAD

**Issue by Am0stafa** [orosmatthew/hydrogen-cpp#26]

**Comment by TheSlugInTub** [comment]
Yes (I think), It was made for a video series and the video series is finished now, I don't think he's gonna update the project or add a new video to the compiler series

**Comment by asdu5asg6da8sjd** [comment]
> Yes (I think), It was made for a video series and the video series is finished now, I don't think he's gonna update the project or add a new video to the compiler series

He hasn't said that he finished the video series, and nanobyte's os series started with 1 hello world video in 2019 (5 years ago) and the second video came out in 2021 (3 years ago), so thats a 2 year wait and the series didnt finish.
